protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2072,3 +2105,53 @@ message GetProtocolUpgradeStatusRequest {}
 message GetProtocolUpgradeStatusResponse {
     bool ready = 1;
 }
+
+
+
+message HistorySegment {
+    int64 from_height = 1;
+    int64 to_height = 2;
+    string chain_id = 3;
+    string history_segment_id = 4;
+    string previous_history_segment_id = 5;
+}
+
+// -- Decentralized History --
+// Decentralised History allows the data-node to reach out to peer nodes to fetch the most recent history, as well as
+// older history if desired, such that it can quickly get itself up to the latest block height of the network and start
+// to consume events for the latest block from the Vega core.
+// See https://github.com/vegaprotocol/vega/tree/develop/datanode/dehistory/README.md for a full description of Decentralised History
```

Comment:
```suggestion
// Decentralized History
//
// Decentralised History allows the data-node to reach out to peer nodes to fetch the most recent history, as well as
// older history if desired, such that it can quickly get itself up to the latest block height of the network and start
// to consume events for the latest block from the Vega core.
// See https://github.com/vegaprotocol/vega/tree/develop/datanode/dehistory/README.md for a full description of Decentralised History
```